documentation/sdk/sidebars.js

```diff
@@ -23,6 +23,16 @@ module.exports = {
       type: 'category',
       label: 'How To',
       items: [
+        {
+          type: "category",
+          label: 'Sign Ed25519',
+          items: [
+            {
+              type: 'autogenerated',
+              dirName: 'how_tos/sign_ed25519',
```

Comment:
No verify?

Reply:
Verify is one of the two examples inside
Should I rename it to `sign_and_verify_ed25519`?

Reply:
Oh, yeah I guess so.